crates/web5/src/crypto/dsa/ed25519.rs

```diff
@@ -37,7 +37,7 @@ pub(crate) fn public_jwk_from_bytes(public_key: &[u8]) -> Result<Jwk> {
     }
 
     Ok(Jwk {
-        alg: "EdDSA".to_string(),
+        alg: "Ed25519".to_string(),
```

Comment:
you sure this is right?
<img width="962" alt="image" src="https://github.com/TBD54566975/web5-rs/assets/5314059/7098f430-6bc7-417d-a266-ae402ffc6b58">

<img width="1454" alt="image" src="https://github.com/TBD54566975/web5-rs/assets/5314059/9a8c50aa-5562-4791-b375-f1e61a49c46d">

I'm a bit confused on this issue at this point, but pretty much all other sources have an ed25519 jwk the alg set to EdDSA

Reply:
yup we're moving to fully-specific alg https://github.com/TBD54566975/web5-rs/issues/128